src/ProjectModule/Components/ProjectList/ProjectList.tsx

```diff
@@ -1,7 +1,127 @@
+  title:title}})
+
+  setPageArray(Array(response.data.totalNumberOfPages).fill().map((_,i)=>i+1));
+
+  console.log(response.data)
+  setProjects(response.data.data); 
+} catch (error) {
+  console.log(error)
+}
+}
+
+useEffect(()=>{ 
+getAllProject(1,5);
+},[])
+// ##################### Filtration ##############################
+const[titleSearch,setTitleSearch]=useState("");
+const getSearchByTitle=(data:any)=>{
+setTitleSearch(data.target.value)
+getAllProject(1,5,data.target.value,titleSearch)
+}
+
+
   return (
-    <div>ProjectList</div>
+    <>
+
+   <div className={`${styles.title} vh-100`} >
+   <div className={`d-flex justify-content-between p-3 bg-white`}>
+      <h3>
+        Projects
+      </h3>
+      <button className="btn-warning rounded-4 btn"><i className="fa-solid fa-plus"></i>Add New Project</button>
+    </div>
+<div className={`p-3 w-75 m-auto ${styles.borderless}`}  >
+  <input type="text"  className="form-control rounded-5 "  onChange={getSearchByTitle} placeholder="Search By Title"/>
+</div>
+    <div className="table p-3 ">
+    {projects.length>0?<table className ="table table-striped caption-top">
+   
+  <thead className={`${styles.bg}`}>
+<tr>
+
+<th className={` ${styles.test2}`} scope="col">#</th>
+            <th className={`${styles.verticalRule }   `} scope="col">Title</th>         
+            <th className={`${styles.verticalRule }   `} scope="col">Description</th>
+            <th className={`${styles.verticalRule }   `} scope="col">TaskNum</th>
+            <th className={` ${styles.test1} ${styles.verticalRule }`} scope="col">Action</th>
+
+
+    </tr>
+</thead>
+ 
+<tbody>
+  {projects.map((pro:any)=>(
+    <tr key={pro.id}  >
+    <th scope="row">{pro.id}</th>
+    <td>{pro.title}</td>
+    <td>{pro.description}</td>
+    <td>{pro.task.length}</td>
+    <td><Dropdown>
+      <Dropdown.Toggle className= {`${styles.solid}`} id="dropdown-basic">
+     
+      </Dropdown.Toggle>
+
+      <Dropdown.Menu>
+        <Dropdown.Item href="#/action-1">  <button   className={`${styles.solid}`} ><i 
+   className='fa fa-edit text-warning mx-2' aria-hidden="true"> </i></button></Dropdown.Item>
+        <Dropdown.Item href="#/action-2"><button   className={`${styles.solid}`}  >
+  <i className='fa fa-trash text-danger mx-2' aria-hidden="true"></i></button></Dropdown.Item>
+      
+      </Dropdown.Menu>
+    </Dropdown></td>
+ 
+  </tr>
+  ))}
+   
+</tbody>
+</table>:<div className="text-center">
+  <img src={noData} className="w-50 "/>
+  </div>}
+    </div>
+    <div className='d-flex justify-content-center bg-white pt-3  '>
+<nav aria-label="Page navigation example">
+  <ul className="pagination">
+    <li className="page-item">
+      <a className={`page-link ${styles.paginationBtn}`} href="#" aria-label="Previous">
+        <span aria-hidden="true">&laquo;</span>
+      </a>
+    </li>
+    {pageArray.map((pageNo,index)=>(
+ <li key={index} className="page-item" onClick={()=>getAllProject(pageNo)}>
```

Comment:
consider adding `aria-label={`go to page${pageNo}`}`, to help screen readers know that this will move them to a certain page number.